### DISCUSSION TITLE
Propuesta de sustituir el menu por iconos o imágenes

### DISCUSSION THREAD

**Discussion by ionhs** [FPGAwars/icestudio#845] [Ideas]
Como sugerencia propongo tener la posibilidad de sustituir el menu de arriba de icestudio por imágenes. Lo comento porque al subir una colección apenas suele haber espacio para clasificar la colección porque el texto ocupa mucho. Por ese motivo se suele crear una doble línea que aunque es funcional queda un poco raro. Tal vez en vez de todo el menu con que se pueda personalizar la clasificación de la colección mediante imágenes podría quedar bien también.

Un saludo y gracias

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
